src/js/utils/dom.js

```diff
@@ -224,3 +224,7 @@ export function openLink(link, target, additionalOptions = {}) {
         a.click();
     }
 }
+
+export function browserIsLandscape() {
```

Comment:
does this exist elsewhere?

Reply:
I wouldn't use `innerWidth` for this.

Reply:
Just in views-manager `onOrientationChange`. It's actually checked differently on older iOS devices using `window.orientation` we have a check in webvr polyfill http://www.williammalone.com/articles/html5-javascript-ios-orientation/

Android
```js
const orientation = window.screen.orientation.type;
const isLandscape = orientation === 'landscape-primary' || orientation === 'landscape-secondary';
```
iOS
```js
const isLandscape = window.orientation == 90 || window.orientation == -90;
``